src/components/tasks/TaskFluency.vue

```diff
@@ -23,12 +24,15 @@ const props = defineProps({
 let TaskLauncher;
 
 const taskId = props.taskId;
+const { version } = packageLockJson.packages['node_modules/@bdelab/roam-fluency'];
 const router = useRouter();
 const gameStarted = ref(false);
 const authStore = useAuthStore();
 const gameStore = useGameStore();
 const { isFirekitInit, roarfirekit } = storeToRefs(authStore);
 
+console.log(version);
```

Comment:
Is this console.log for development or do we intend to keep it in production?

Reply:
Don't need it, thanks!